lib/comparing.dart

```diff
@@ -10,7 +10,7 @@ class Comparing {
 
   /// Returns a Comparator that compares objects of type T by mapping them to
   /// Comparables.
-  static Comparator<T> on<T>(Comparable Function(T t) mapper) {
+  static Comparator<T> on<T>(Comparable<Object> Function(T t) mapper) {
```

Comment:
I think it has to be `Object?` to not be a breaking change

Reply:
Agreed. Do you want me to do `Object?` then? Or do we want to be more strict and disallow nullable comparables?

Reply:
I think `Object?` should be fine

Reply:
Done.